sbin/xivo-check-master-status

```diff
@@ -9,7 +9,11 @@ if [ -z "$ip_addr" ]; then
 fi
 
 ping -c $ping_count -i $ping_interval $ip_addr
-if [ $? -eq 0 ]; then
+ping_status=$?
+if [ ! -f /var/lib/wazo/is-secondary ]; then
+    echo "HA mode changed, this stack is no longer a secondary" >&2
+    exit 0
+else if [ $ping_status -eq 0 ]; then
```

Comment:
Oh thanks to the debian-packaging error
Lintian has detected this invalid syntax :smile:

Reply:
Oups my bad, should've linted/test it.